src/components/ThumbnailImage.js

```diff
@@ -19,26 +19,42 @@ const propTypes = {
     /** Do the urls require an authToken? */
     isAuthTokenRequired: PropTypes.bool.isRequired,
 
+    /** Image size */
+    imageWidth: PropTypes.number,
+    imageHeight: PropTypes.number,
+
```

Comment:
Add separate comments for each prop. 
`Width of the thumbnail image`

`Height of the thumbnail image`

Reply:
Added.